src/line_info.c

```diff
@@ -20,6 +20,10 @@
  *
  */
 
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE    /* needed for wcswidth() */
```

Comment:
`man 3 wcswidth` say it's `_XOPEN_SOURCE`, but I guess either works?

Reply:
I copy pasted it from somewhere else in toxic, so it works and I don't want to temp fate by changing defines.